EpikCord/abstract.py

```diff
@@ -347,11 +347,16 @@ async def discover_ip(self):
 
 
 class GuildChannel(BaseChannel):
-    def __init__(self, client, data: dict):
+    def __init__(self, client, data: Union[
+        discord_typings.VoiceChannelData,
+        discord_typings.TextChannelData,
+        discord_typings.CategoryChannelData,
+        discord_typings.NewsChannelData,
+    ]):
         super().__init__(client, data)
-        self.guild_id: str = data.get("guild_id")
-        self.position: int = data.get("position")
-        self.nsfw: bool = data.get("nsfw")
+        self.guild_id: int = int(data["guild_id"])
+        self.position: int = data["position"]
+        self.nsfw: bool = data["nsfw"]
         self.permission_overwrites: List[dict] = data.get("permission_overwrites")
```

Comment:
type that with Optional[]

Reply:
Wasn't *really* the scope of things, but will do